reactjs-jsx-ppx/reactjs_jsx_ppx_v3.ml

```diff
@@ -540,9 +538,8 @@ let jsxMapper =
                   Ocaml_common.Location.prerr_warning pattern.ppat_loc
                     (Preprocessor
                        (Printf.sprintf
-                          "ReasonReact: optional argument annotations must \
-                           have explicit `option`. Did you mean \
-                           `option(%s)=?`?"
+                          "React: optional argument annotations must have \
```

Comment:
Maybe we could refer to the ppx name to make it clearer where the error is coming from, as there are a few parts involved when using React components in Reason:
- the ReactJS bindings
- the Reason parser JSX transformation
- and the reactjs-jsx-ppx preprocessor

e.g.

```suggestion
                          "reactjs-jsx-ppx: optional argument annotations must have \
```